www/class/centreon-clapi/centreonDowntime.class.php

```diff
@@ -544,7 +662,8 @@ public function addservice($parameters)
                 AND service_service_id = ?";
             $stmt = $this->db->query($sql, array($downtimeId, $ids[0], $ids[1]));
             if ($stmt->rowCount()) {
-                throw new CentreonClapiException(sprintf('Relationship with %s / %s already exists', $host, $service));
+                throw new CentreonClapiException(sprintf('Relationship with %s / %s already exists', $host,
+                    $service));
```

Comment:
Those kind of multiline must be 

throw new CentreonClapiException(sprintf(
    'Relationship with %s / %s already exists',
    $host,
    $service
));